app/styles/sub-styles/filter-nav.scss

```diff
@@ -1,31 +1,40 @@
-// Nav-3 section 
+// Nav-3 section
 
 .filter-by {
   display: flex;
   height: 20px;
   margin: 0 0 10px 0;
   cursor: pointer;
+  float: left;
+
 }
 
+.filter-by::after{
+  content: none;
```

Comment:
Why is `content: none` required here..?

Reply:
This for removing default dropdown arrow.